dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/Intrinsics.java

```diff
@@ -1565,6 +1576,15 @@ private List<Event> inlineMemSet(FunctionCall call) {
         return replacement;
     }
 
+    private List<Event> inlineLLVMThreadLocal(FunctionCall call) {
+        final Register resultReg = getResultRegisterAndCheckArguments(1, call);
+        final Expression exp = call.getArguments().get(0);
+        checkArgument(exp instanceof MemoryObject object && object.isThreadLocal(), "Calling thread local instrinsic on a non thread local object \"%s\"", call);
```

Comment:
Typo. Should be "Calling thread-local intrinsic on a non-thread-local object ..."